frontend/src/components/RegistrationComplete/RegistrationComplete.jsx

```diff
@@ -31,13 +33,15 @@ const Button = styled.button`
   font: Urbanist;
 `;
 
-export default function RegistrationComplete() {
+export default function RegistrationComplete({ selectedEvent }) {
   return (
     <div>
       <Container>
         <h1>Registered For</h1>
       </Container>
-      <div>{/* EventCard placeholder here */}</div>
+      <div>
+        <EventCard event={selectedEvent} />
+      </div>
```

Comment:
Could you add a bit more styling to this page so it looks like the mockups? It currently looks like this on my computer:
![image](https://user-images.githubusercontent.com/32073638/153960338-b1176eab-83fe-4675-978a-6ad2c597156a.png)